core/services/pipeline/runner_test.go

```diff
@@ -23,8 +23,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// TODO: Add a test for multiple terminal tasks after __result__ is deprecated
```

Comment:
Did you add the test?

Reply:
this test has multiple terminal tasks after removing the __result__ task